admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1197,7 +1197,7 @@ $product-page-padding-bottom: 80px !default;
     }
   }
 
-  .product-tabs {
+  #product-tabs-content {
```

Comment:
I don't see this id anywhere in the code :thinking: 
Is it created dynamically ?

Reply:
Yep exactly, it's built by the navigation component in the form theme https://github.com/PrestaShop/PrestaShop/blob/f4187be937896eede5188e52f9b747d2e5dddf44/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L1420